Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -439,6 +439,61 @@ public IEnumerator TestDestroy()
             Assert.AreEqual(propBlock.GetColor("_Color"), FocusColor);
         }
 
+        [UnityTest]
+        /// <summary>
+        /// Tests button depth and focus state after enabling, disabling and re-enabling Interactable 
+        /// internally via SetDisable(true). The focus state after re-enabling should be false and button
+        /// depth should be in its default position.  This test is specifically addressing behavior described 
+        /// in issue 4967.
+        /// </summary>
+        public IEnumerator TestFocusStateOnDisableEnableInteractable()
```

Comment:
Rename to slightly simpler: TestInteractableDisableOnClick
or anything else shorter if you prefer